adoc/chapters/programming_interface.adoc

```diff
@@ -6253,7 +6254,7 @@ There are several different {cpp} classes that implement accessors:
 
 Accessor objects must always be constructed in host code, either in
 <<command-group-scope>> or in <<application-scope>>.  Whether the constructor
-blocks waiting for data to synchronize depends on the type of accessor.  Those
+blocks until data has been copied back depends on the type of accessor.  Those
```

Comment:
```suggestion
blocks depends on the type of accessor.  Those
```

Can we just say this here?  I'd rather not say "copied back" because some implementations of accessors will not require memory to be copied (e.g. USM).  Note below that we say it blocks until "the data is available".

Reply:
I have a weak preference for giving some indication of what condition will unblock the call.  Could we repeat "blocks until data is available" here?

Reply:
Yes, that sounds good.

Reply:
Fixed in 9bcbefb40be659fa1a8ce05c0551801188e0e0b9.